ArduCopter/Copter.h

```diff
@@ -28,18 +28,18 @@
 #include <AP_HAL/AP_HAL.h>
 
 // Common dependencies
-#include <AP_Common/AP_Common.h>
-#include <AP_Common/Location.h>
-#include <AP_Param/AP_Param.h>
-#include <StorageManager/StorageManager.h>
+#include <AP_Common/AP_Common.h>        //Common definitions and utility routines for the ArduPilot libraries.
```

Comment:
Nitpicks.
Probably should add a space between all of these `// `. 
Also you won't see many periods unless it is a full sentence
```suggestion
#include <AP_Common/AP_Common.h>        // Common definitions and utility routines for the ArduPilot libraries
```
Usually, all of the comments get lined up together which it looks like you did with most of your changes.  :)
Pre-existing of course, but if you got adventurous you could line up the rest.